.github/workflows/tests.yml

```diff
@@ -44,15 +44,18 @@ jobs:
         run: |
           sudo apt install -y libgmp3-dev
           python -m pip install --upgrade pip
-          pip install starknet-devnet==0.1.23
+          pip install starknet-devnet==0.1.19
```

Comment:
any particular reason as to why we're downgrading starknet-devnet ?

Reply:
it doesnt work with anything more recent in the GA so its sort of rough fix for now. super annoying